CHANGELOG.md

```diff
@@ -126,6 +126,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - **CUMULUS-2309**
     - Changed `DELETE /granules/{granuleName}` API behavior to also delete record from Postgres database.
     - Changed `Bulk operation BULK_GRANULE_DELETE` API behavior to also delete records from Postgres database.
+  - **CUMULUS-2367**
+    - Updated `granule_cumulus_id` foreign key to granule in RDS `files` table to use a CASCADE delete, so records in the files table are automatically deleted by the database when the corresponding granule is deleted.
```

Comment:
This is def a nit and kind of unrelated to your PR but I am wondering if we should consolidate to determine whether or not we should refer to it as `RDS` or `Postgres` because I've seen both in the changelog

Reply:
Also it's fine it it doesn't matter as well

Reply:
We should definitely agree/fix.   I think Postgres is more appropriate as Core isn't tied to RDS specifically

Reply:
I just updated it throughout the CHANGELOG

dd42cdb5f0bbd0ee5222263d8be5724a99d9cf50